src/AbstractActionReader.ts

```diff
@@ -1,5 +1,15 @@
 import * as Logger from "bunyan"
-import { ActionReaderConfig, Block, BlockInfo, BlockMeta, NextBlock } from "./interfaces"
+import { ActionReaderOptions, Block, BlockInfo, BlockMeta, NextBlock } from "./interfaces"
+
+const defaultBlock = {
```

Comment:
Any to cast here instead of `const defaultBlock: Block = {...}`?

Reply:
The main reason is because I reference this elsewhere (lines 89, 220, 224), and besides that I think it looks nicer.